api/src/main/java/com/velocitypowered/api/util/ModInfo.java

```diff
@@ -62,19 +64,23 @@ public static final class Mod {
 
     @SerializedName("modid")
     private final String id;
-    private final String version;
+    private final @Nullable String version;
 
-    public Mod(String id, String version) {
+    public Mod(String id) {
+      this(id, null);
+    }
+
+    public Mod(String id, @Nullable String version) {
       this.id = Preconditions.checkNotNull(id, "id");
-      this.version = Preconditions.checkNotNull(version, "version");
+      this.version = version;
     }
 
     public String getId() {
       return id;
     }
 
-    public String getVersion() {
-      return version;
+    public Optional<String> getVersion() {
```

Comment:
This is a breaking API change. I would probably just return an empty string in this case.